lib/questionnaires/model/src/questionnaire_item_model.dart

```diff
@@ -1,6 +1,7 @@
-import 'dart:core';
+import 'dart:ui';
```

Comment:
If we do the other suggestion, we don't need to switch to `dart:ui` anymore I think.